packages/geo/src/stac/stac.attribution.ts

```diff
@@ -21,7 +21,7 @@ export type AttributionItem = StacItem<{
 /**
  * A Single File STAC compliant attribution json structure.
  */
-export interface AttributionStac extends StacCatalog, GeoJSON.GeoJsonObject {
+export interface AttributionStac extends StacCatalog, GeoJSON.GeoJsonObject, Record<string, unknown> {
```

Comment:
This was complaining about being cast to a `Record<string, unknown>` not sure if there is a better way around this